apps/ledger-live-mobile/src/screens/CustomError/index.tsx

```diff
@@ -28,10 +28,19 @@ export default function CustomError({ route }: CustomErrorPropsProps) {
         <Text variant="body" textAlign={"center"}>
           {t("errors.CustomError.description")}
 
-          <Trans
-            i18nKey="errors.CustomError.errorCode"
-            values={{ errorCode: error?.cause.swapCode }}
-          />
+          {error && "cause" in error && error.cause?.swapCode && (
+            <Trans
+              i18nKey="errors.CustomError.errorCode"
+              values={{ errorCode: error.cause.swapCode }}
+            />
+          )}
+
+          {error && "message" in error && error.message && (
```

Comment:
Hello @KVNLS.  Thanks for the suggestion, but if i do optional chaining, i get an error:

```
Property 'cause' does not exist on type 'SwapLiveError | Error'.
  Property 'cause' does not exist on type 'Error'.ts(2339)
  ```
  
 The error happens because error can be of type Error, which doesn't have a cause property. While optional chaining can prevent runtime errors when accessing potential undefined properties, it doesn't help TS recognize properties that don't exist on certain types.
 
 I use "cause" in error to narrow the type, letting TS that error has a cause property before accessing it, this way, TS knows it's safe to access error.cause